common/math/geometry/src/bounding_box.cpp

```diff
@@ -151,6 +152,55 @@ std::vector<geometry_msgs::msg::Point> getPointsFromBbox(
   return points;
 }
 
+auto get2DPolygon(const traffic_simulator_msgs::msg::BoundingBox & bounding_box)
+  -> std::vector<geometry_msgs::msg::Point>
+{
+  std::vector<geometry_msgs::msg::Point> points_bbox;
```

Comment:
I noticed that `bbox` is commonly used, so I decided to use it as well. 
Thank you so much for your comment, I've changed all `bbox` I introduced to `bounding_box`. 
I also changed it in a couple more parts, but not all of them - because I didn't want to increase the PR so much, that is already large.